<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>简单的ComboBox</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>初始化列表</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>清除列表</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>可编辑</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>北京市</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/images/Lighthouse.jpg</normaloff>:/images/images/Lighthouse.jpg</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>上海市</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>321</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>有用户数据的ComboBox</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>初始化城市+区号</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBox_2">
      <item>
       <property name="text">
        <string>北京</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>上海</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>611</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>QPlainTextEdit具有标准快捷键菜单</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>将文本内容添加到ComboBox</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>清除文本内容</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>Read Only</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
